packages/playground/website/src/components/site-manager-sidebar/index.tsx

```diff
@@ -0,0 +1,232 @@
+					className={css.siteManagerSidebarLogoButton}
+					onClick={onLogoClick}
+				>
+					{/* TODO: move logo to component package */}
+					<svg
+						width="32"
+						height="32"
+						viewBox="0 0 32 32"
+						fill="none"
+						xmlns="http://www.w3.org/2000/svg"
+					>
+						<rect
+							width="10.4176"
+							height="10.4176"
+							rx="3.86258"
+							transform="matrix(0.829038 -0.559193 0.838671 0.544639 7.45703 24.1775)"
+							stroke="white"
+							strokeWidth="0.965644"
+						/>
+						<rect
+							width="13.2346"
+							height="13.2346"
+							rx="3.86258"
+							transform="matrix(0.829038 -0.559193 0.838671 0.544639 5.0918 18.9934)"
+							stroke="white"
+							strokeWidth="1.44847"
+						/>
+						<rect
+							width="17.451"
+							height="17.451"
+							rx="3.86258"
+							transform="matrix(0.829038 -0.559193 0.838671 0.544639 1.55371 11.6099)"
+							stroke="white"
+							strokeWidth="1.93129"
+						/>
+					</svg>
+				</button>
+			</header>
+			<div
+				className={classNames(
+					css.siteManagerSidebarSection,
+					css.siteManagerSidebarContent
+				)}
+			>
+				<h2 className={css.siteManagerSidebarSubtitle}>
+					WordPress Playground
+				</h2>
+				<label
+					className={classNames(
+						css.siteManagerSidebarLabel,
+						css.siteManagerSidebarListLabel
+					)}
+					htmlFor="site-list"
+				>
+					Your sites
+				</label>
+				<ul id="site-list" className={css.siteManagerSidebarList}>
+					{sites.map((site) => (
+						<li
+							key={site.slug}
+							className={css.siteManagerSidebarItem}
+						>
+							<button
+								className={css.siteManagerSidebarItemButton}
+								onClick={() => onSiteChange(site.slug)}
+							>
+								<img
+									src={getLogoDataURL(site.logo)}
+									alt="Site logo"
+									className={css.siteManagerSidebarItemLogo}
+								/>
+								<span
+									className={
+										css.siteManagerSidebarItemSiteName
+									}
+								>
+									{site.name}
+								</span>
+								{(site.storage === 'none' || !site.storage) && (
+									<span
+										className={
+											css.siteManagerSidebarItemStorageIcon
+										}
+										title="No storage"
+									>
+										{/* TODO: move icon to component package */}
+										<svg
+											width="16"
+											height="17"
+											viewBox="0 0 16 17"
+											fill="none"
+											xmlns="http://www.w3.org/2000/svg"
+										>
+											<path
+												fillRule="evenodd"
+												clipRule="evenodd"
+												d="M8 15C4.41015 15 1.5 12.0899 1.5 8.5C1.5 4.91015 4.41015 2 8 2C11.5899 2 14.5 4.91015 14.5 8.5C14.5 12.0899 11.5899 15 8 15ZM0 8.5C0 4.08172 3.58172 0.5 8 0.5C12.4183 0.5 16 4.08172 16 8.5C16 12.9183 12.4183 16.5 8 16.5C3.58172 16.5 0 12.9183 0 8.5ZM9 9.5V4.5H7.5V8H5.5V9.5H9Z"
+												fill="#949494"
+											/>
+										</svg>
+									</span>
+								)}
+							</button>
+						</li>
+					))}
+				</ul>
+			</div>
+			<footer
```

Comment:
Can we use the WordPress flex component for this like in the site editor? Or would that require updating the components library package?